readme.md

```diff
@@ -23,17 +23,29 @@ const Emittery = require('emittery');
 
 const emitter = new Emittery();
 
+const myEvent = Symbol('my symbol event');
+
 emitter.on('🦄', data => {
-	console.log(data);
-	// '🌈'
+	console.log(data); // '🌈'
+});
+
+emitter.on(myEvent, data => {
+	console.log(data); // '🦋'
 });
 
 emitter.emit('🦄', '🌈');
+emitter.emit(myEvent, '🦋')
+
```

Comment:
Can you add this example to the TS file too?

Reply:
We don't have generic usage example in typescript, only examples to clarify some non-obvious things.